rust/arrow-flight/src/arrow.flight.protocol.rs

```diff
@@ -498,9 +498,8 @@ pub mod flight_service_server {
     #[async_trait]
     pub trait FlightService: Send + Sync + 'static {
         #[doc = "Server streaming response type for the Handshake method."]
-        type HandshakeStream: Stream<
-            Item = Result<super::HandshakeResponse, tonic::Status>,
-        > + Send
+        type HandshakeStream: Stream<Item = Result<super::HandshakeResponse, tonic::Status>>
```

Comment:
There might be a `rustfmt` version mismatch that's going to have us changing this back and forth. I've also seen rustfmt suggest changing this, but I've been ignoring it so far. What version of `rustfmt` are you using @Dandandan ?

Reply:
I've been seeing this locally too, but not all the time

Reply:
Me too

Reply:
Will have a look. I saw it being changed during a PR indeed, thought it had to do with a recent rust upgrade. Seems that the CI accepts both alternatives, so I guess it's a newer version doing more formatting.

Reply:
Looks like I probably was using a more recent version indeed and have it on the nightly version by default:
`rustfmt 1.4.29-nightly (70ce182 2020-12-04)`

Reply:
FWIW I removed the changes to `rust/arrow-flight/src/arrow.flight.protocol.rs` from this PR.